packages/cli/commands/sandbox/delete.js

```diff
@@ -44,6 +44,8 @@ exports.handler = async options => {
   const { account, force } = options;
   const config = getConfig();
 
+  trackCommandUsage('sandbox-delete', null);
```

Comment:
Previously we were not tracking this command if it exited early. Moved this to the top of the command